application/src/main/java/net/corda/v5/application/crypto/DigitalSignatureVerificationService.java

```diff
@@ -40,7 +38,7 @@ public interface DigitalSignatureVerificationService {
      * @param signatureSpec The signature spec.
      *
      * @throws CryptoSignatureException If verification of the digital signature fails.
-     * @throws IllegalArgumentException If the signature scheme is not supported or if any of the clear or signature
+     * @throws IllegalArgumentException If the signature scheme is not supported or if any of the original or signature
      * data is empty.
```

Comment:
nitpicky: can we please align indentation here to be as above?

Reply:
Fixed.